openlibrary/templates/search/sort_options.html

```diff
@@ -38,34 +36,37 @@
        ]
        if ctx.user and (ctx.user.is_admin() or ctx.user.is_librarian()):
          sort_options.append({ 'sort': 'title', 'name': _("Work Title (beta: Librarian only)"), 'ga_key': 'Title' })
-    $for sort_option in sort_options:
-      $if exclude and sort_option['sort'] in exclude:
-        $continue
-      $ is_selected = sort_option.get('selected') or sort_option['sort'] == selected_sort or (selected_sort is None and sort_option['sort'] == default_sort)
-      $if is_selected:
-          <a><strong>$sort_option['name']</strong></a>
-      $else:
-          <a href="$changequery(page=None, sort=cond(sort_option['sort'] == default_sort, None, sort_option['sort']))"
-             data-ol-link-track="SearchSort|$sort_option['ga_key']"
-             rel="nofollow"
-          >$sort_option['name']</a>
-      $if sort_option.get('sub_sorts') and is_selected:
-        <select
-          data-ol-link-track="SearchSort|$sort_option['ga_key']SubSort"
-          onchange="window.location.href = this.value"
-        >
-          $for sub_sort in sort_option['sub_sorts']:
-            $ is_selected = sub_sort['sort'] == selected_sort
-            <option
-              value="$changequery(page=None, sort=cond(sub_sort['sort'] == default_sort, None, sub_sort['sort']))"
-              $cond(is_selected, 'selected')
-            >$sub_sort['name']</option>
-        </select>
-      $if not loop.last:
-        |
+    <span class="sort-dropper">
+      <details>
+        <summary>
+          <button class="sort-btn">Sort Options</button>
```

Comment:
Looks like you're also not quite passing the `detect-missing-i18n` check, this just confirms that any text added to the site is [prepped for internationalization](https://github.com/internetarchive/openlibrary/wiki/Internationalization#internationalization-i18n-developers-guide). 

In this case, all you need to do is add the standard HTML onscreen text formatting, like so:
```suggestion
          <button class="sort-btn">$_("Sort Options")</button>
```

Reply:
Wow, you predicted what I was going to ask!

Reply:
Woohoo! Yeah, it's a great new script by @pidgezero-one that should help us catch a lot of untranslated text, and help give newer contributors an easy intro to the internationalization process.